src/javascript/app/pages/trade/tradepage.js

```diff
@@ -57,6 +59,32 @@ const TradePage = (() => {
                     if ($currency.next().attr('id') === $currency.attr('id')) $currency.next().eq(0).remove();
                 }
             });
+
+            BinarySocket.wait('website_status', 'landing_company').then(() => {
+                const is_logged_in = Client.isLoggedIn();
+                const is_eu = isEuCountry();
+                const is_maltainvest = Client.get('landing_company_shortcode') === 'maltainvest';
+                const show_msfa_message = () => {
+                    $('.mfsa_message')
+                        .removeClass('container')
+                        .addClass('margin-bottom-40')
+                        .slideDown(300);
+                };
+                
+                if (is_eu) {
+                    if (is_logged_in && is_maltainvest) {
+                        BinarySocket.wait('get_account_status').then((response) => {
+                            const is_professional = (getPropertyValue(response, ['get_account_status', 'status']) || []).includes('professional');
+                            if (!is_professional) {
+                                $('#professional-cta').setVisibility(1);
```

Comment:
I suggest using `client_logged_in` class name instead of this ID for the CTA element so we can remove this line and let the common code handle showing/hiding the CTA

Reply:
Could you elaborate? The _client\_logged\_in_ class should show elements for clients that are logged in, in this case we only want to show the CTA to logged in, non-professionals, if we would use the _client\_logged\_in_ it would still show for logged in professionals?